README.md

```diff
@@ -72,8 +72,8 @@ Allowed properties :
   - `true`: Continues even if they are errors (**default for now**)
   - `false`: Stops if an error occurs
 - `elapsedRecordField`: Boolean
-  - `true`: Includes a `elapsed_time` field inside each `record` field, containing the elapsed time in seconds since the first record (**default**)
-  - `false`
+  - `true`: Includes `elapsed_time`, containing the elapsed time in seconds since the first record, and `timer_time`, containing the time shown on the device, inside each `record` field
+  - `false` (**default**)
```

Comment:
It looks like `false` [is the default](https://github.com/pierremtb/easy-fit/blob/e5dc98f9d8a9671d4a92d5392038b9338cb309ed/src/easy-fit.js#L10).